docs/tutorials/mastering.md

```diff
@@ -694,75 +785,111 @@ Log out of the VM.
 
 ### Prepare SD-Core Control Plane VM
 
-Login to the `control-plane` VM.
+Login to the `control-plane` VM:
 
 ```console
 multipass shell control-plane
 ```
 
-Install Microk8s.
+Install MicroK8s:
+
+```console
+sudo snap install microk8s --channel=1.28-strict/stable
```

Comment:
Are we going to 1.29 or 1.28?

Reply:
For some reason I thought we were going for 1.28... But you're right, it's 1.29.
Updated.